db/db.go

```diff
@@ -20,6 +20,10 @@ func InitDB() {
 }
 
 func constructDsn() string {
+	if os.Getenv("APP_ENV") == "release" {
```

Comment:
I think later we should manage this with an extra layer, somehow I felt that db init and app build type are very close to each other, and both are manageable by "APP_ENV",  but yet we are managing their classes separately 🤔  just a bit of inconvenient since the same check is duplicated and the same logic is fragmented imo. We can discuss later off this PR

Reply:
Good catch @sleepylee, Should I refactor this in this PR? If yes how about something like
```go
func SetupApp() {
   env := os.Getenv("APP_ENV")

   switch env {
	case ReleaseMode:
		// does release setup (load envs, database setup and so on)
	case TestMode:
		// does test setup (load envs, database setup and so on)
	default:
		// does development setup (load envs, database setup and so on)
	}
}
```

Reply:
I think we can refactor this in another PR.